src/main/java/org/hobbit/core/components/AbstractCommandReceivingComponent.java

```diff
@@ -554,5 +562,28 @@ public void close() throws IOException {
         }
         super.close();
     }
+    
+    public String getContainerName() {
+		return containerName;
+	}
+
+	public void setContainerName(String containerName) {
+		this.containerName = containerName;
+	}
+	
+	public Gson getGson() {
+		return gson;
+	}
+
+	public void setGson(Gson gson) {
+		this.gson = gson;
+	}
+
+	public void createDummyComponent(byte command, byte[] data, String sessionId, BasicProperties props) {
+		// TODO Auto-generated method stub
+		
+	}
```

Comment:
Didn't we discuss that this method is not necessary in this abstract class? Or is there a reason why it is still here?